### PR TITLE
Feat/add ability to clone note

### PR DESCRIPTION
**Pull request by Antogin**
Hi, this is my proposition for issue #1437.

I'm new to open source and very open to feed back. Be harsh plz :smiley:

### CODE REVIEW

File: browser/main/NoteList/index.js [hunk 1]
```diff
@@ -543,6 +548,44 @@ class NoteList extends React.Component {
     this.setState({ selectedNoteKeys: [] })
   }
 
+  cloneNote () {
+    const { selectedNoteKeys } = this.state
+    const { dispatch, location } = this.props
+    const { router } = this.context
```
Comment (sosukesuzuki) [inline browser/main/NoteList/index.js]: please remove `router`. It is never used.

Comment (Antogin) [reply browser/main/NoteList/index.js]: resolved

**Comment by sosukesuzuki** [comment]
Sorry, our travis is broken...:bow:

**Comment by kazup01** [comment]
Merged. Thanks for the amazing support @Antogin :)

**Pull request merged**